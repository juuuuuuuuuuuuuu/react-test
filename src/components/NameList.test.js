import React from 'react';
import renderer from 'react-test-renderer';
import NameList from './NameList';

describe('NameList', () => {
  //초기 렌더링 
  let component = null;

  it('renders correctly', () => {
    component = renderer.create(<NameList names={["이주영", "이주영1"]}/>);
  })

  it('matches snapshot', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})