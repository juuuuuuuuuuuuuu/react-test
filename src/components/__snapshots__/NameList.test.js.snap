// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameList matches snapshot 1`] = `
<ul>
  <li>
    이주영
  </li>
  <li>
    이주영1
  </li>
</ul>
`;
